dandiapi/api/views/info.py

```diff
@@ -23,14 +23,23 @@ class ApiServicesSerializer(serializers.Serializer):
 
 
 class ApiInfoSerializer(serializers.Serializer):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        # Need to add fields here to allow for hyphens in name
+        self.fields.update(
+            {
+                'cli-minimal-version': serializers.CharField(),
+                'cli-bad-versions': serializers.ListField(child=serializers.CharField()),
```

Comment:
In addition to the test, I think we should have a comment somewhere here saying that the CLI expects these exact keywords. Someone who doesn't know that could come along and change it + update the test without being aware of it.